courses/serializers_test.py

```diff
@@ -0,0 +1,114 @@
+"""
+Tests for course serializers
+"""
+# pylint: disable=unused-argument, redefined-outer-name
+import pytest
+
+from courses.factories import ProgramFactory, CourseFactory, CourseRunFactory
+from courses.serializers import ProgramSerializer, CourseSerializer, CourseRunSerializer
+
+
+def test_serialize_program():
+    """Test Program serialization"""
+    program = ProgramFactory.build()
```

Comment:
Could you switch this and the other tests to `create()` so that `id`, `created_on`, and `updated_on` are tested?